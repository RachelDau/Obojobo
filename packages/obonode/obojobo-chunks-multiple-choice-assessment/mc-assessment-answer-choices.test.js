jest.mock(
	'Common',
	() => ({
		page: {
			focus: jest.fn()
		}
	}),
	{ virtual: true }
)
import React from 'react'
import renderer from 'react-test-renderer'
import MCAssessmentAnswerChoices from './mc-assessment-answer-choices'

let defaultProps

describe('MCAssessment Answer Choices', () => {
	beforeEach(() => {
		defaultProps = {
			responseType: '',
			score: '',
			correctLabel: '',
			incorrectLabel: '',
			pickAllIncorrectMessage: '',
			models: [],
			moduleData: '',
			mode: ''
		}
	})

	test('Renders', () => {
		const component = renderer.create(<MCAssessmentAnswerChoices {...defaultProps} />)
		const tree = component.toJSON()
		expect(tree).toMatchSnapshot()
	})
})
