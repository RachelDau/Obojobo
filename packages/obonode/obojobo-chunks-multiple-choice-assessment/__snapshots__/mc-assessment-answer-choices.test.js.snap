// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MCAssessment Answer Choices focusOnResults renders 1`] = `
<div
  role="radiogroup"
/>
`;
